src/Serializable.ts

```diff
@@ -165,10 +217,24 @@ export class Serializable extends SerializableBase implements ISerializable {
         return this.postFrom(realObj)
     }
 
+    /**
+     * `preFrom` can be overwritten to manipulate the value before the parsing.
+     * This allows to add logic to the deserialization without having to override the fromAny method.
+     *
+     * @param value the object that will be manipulated before the actual parsing.
+     * @returns the manipulated object
+     */
     protected static preFrom(value: any): Promise<any> | any {
         return value
     }
 
+    /**
+     * `postFrom` can be overwritten to manipulate the value after the parsing.
+     * This allows to add logic to the deserialization without having to override the fromAny method.
```

Comment:
This allows to add logic to the **parsing** without having to override the fromAny method.